lib/libnvpair/libnvpair.suppr

```diff
@@ -1,2 +1,5 @@
 [suppress_type]
 	name = FILE*
+
+[suppress_function]
+	name = libspl_assertf
```

Comment:
Since we intentionally don't maintain a stable `.abi` file for `libspl` I'm not sure why this was needed.  Did the CI style checkers flag this?